### ISSUE TITLE
Migrate to the Web Design Standards (or 18F brand?)

### ISSUE THREAD

**Issue by shawnbot** [18F/tock#478]
We're currently using a very minimal stylesheet based on Bourbon and Neat. Making stylistic and/or behavioral changes is going to be tricky without a coherent design framework, so I think it would be good to work off a base of the [Web Design Standards](https://standards.usa.gov/) (and/or the [18F brand guidelines](https://pages.18f.gov/brand/)).

Dependencies:
- [x] Merge #437 to fix #436.

**Comment by shawnbot** [comment]
@abrouilette I'd like to raise this as an important next step. @maya has done some prototyping of the site with both the [Standards](https://standards.usa.gov) and the [18F brand styles](https://pages.18f.gov/brand/), and they're gorgeous. (Maya, feel free to paste those here!) Having the _framework_ of the Standards in place will make it much simpler and easier to solve other problems, like screen size responsiveness in #480, and making the hours input more user-friendly in #424.

**Comment by abrouilette** [comment]
@shawnbot, @maya, what kind of lift is that, time-wise?  Also, is it work that can/should be done before the research starts?

**Comment by maya** [comment]
Here are the prototypes I created:

## Web Design Standards (straight up)
<img width="1235" alt="screen shot 2016-09-22 at 4 40 31 pm" src="https://cloud.githubusercontent.com/assets/5249443/18976339/06a3b9ea-8666-11e6-9e5b-163345b9edbf.png">


## Web Design Standards (18F-ified)
<img width="1240" alt="screen shot 2016-09-22 at 4 56 48 pm" src="https://cloud.githubusercontent.com/assets/5249443/18976344/0a96d7d0-8666-11e6-8127-d8f993f3fcd0.png">

I don't think it will be too heavy a lift (a few days?), and could possibly be done alongside the research or at the beginning of the project.

**Comment by abrouilette** [comment]
If @maya and @shawnbot are aligned, what business have I standing in the way?  Fire it off.

**Comment by maya** [comment]
Some great mockups by @jenniferthibault!

We will likely wait until the research is complete before revisiting.

## Landing page
![tock-landing](https://cloud.githubusercontent.com/assets/5249443/19527406/c2ee7060-95dc-11e6-892e-e66fbd5b6825.jpg)

## Timecard
![tock-timecard](https://cloud.githubusercontent.com/assets/5249443/19527411/c6bd03dc-95dc-11e6-8ee8-42fbf9a00d0a.jpg)

**Comment by abrouilette** [comment]
@maya, regarding what @shawnbot said above ("Having the framework of the Standards in place will make it much simpler and easier to solve other problems, like screen size responsiveness in #480, and making the hours input more user-friendly in #424."), should we be splitting this issue into two or more issues?  One for getting the framework in place, and another for the design work?  Or do I not understand "framework" in this context.  (Which is likely.)

**Comment by maya** [comment]
@abrouilette yes, adding the framework and creating visual consistency with 18F brand and UX improvements can be their own issues. I'll add a new issue for the latter so we can keep this issue for getting the foundation setup.
